src/scsiencrypt.h

```diff
@@ -45,7 +45,7 @@ constexpr uint8_t RDMC_DEFAULT = 0x00;
 
 // outputs hex in a 2 digit pair
 #define HEX(x)                                                                 \
-    std::right << std::setw(2) << std::setfill('0') << std::hex << (int)(x) << std::setfill(' ')
+    std::right << "0x" << std::setw(2) << std::setfill('0') << std::hex << (int)(x) << ", " << std::setfill(' ')
```

Comment:
@ninthclowd  do I remember correct, that you wrote once that people rely in scripts on this output? I have no objections to the change

Reply:
Same as above - this was to help capture the traffic for setting up the tests and should be stripped out. I do like putting spaces in the output though which helps when staring at the bytes for debugging.